flutter_appauth/android/src/main/java/io/crossingthestreams/flutterappauth/FlutterAppauthPlugin.java

```diff
@@ -333,6 +336,12 @@ private void performAuthorization(AuthorizationServiceConfiguration serviceConfi
                 authRequestBuilder.setUiLocales(additionalParameters.get("ui_locales"));
                 additionalParameters.remove("ui_locales");
             }
+
+            if(additionalParameters.containsKey("claims")){
+                final JSONObject claimsAsJson = new JSONObject(additionalParameters.get("claims"));
```

Comment:
Looks like the pipeline indicates that Android app is failing to build though due to a potential unhandled `JSONException`. Would it be possible to catch this and return an error over to the Dart side? How this is done elsewhere within the Java code is by calling the `finishWithError` method.

Reply:
Thanks for the hint - updated accordingly. :-)

Reply:
Can also confirm that using `additionalParameters` on iOS works like a charm.